astropy/tests/runner.py

```diff
@@ -191,24 +375,47 @@ def run_tests(self, package=None, test_path=None, args=None, plugins=None,
                           AstropyDeprecationWarning)
             remote_data = 'any'
 
-        all_args.append('--remote-data={0}'.format(remote_data))
+        return ['--remote-data={0}'.format(remote_data)]
 
+    @keyword()
+    def pep8(self, pep8, kwargs):
+        """
+        pep8 : bool, optional
+            Turn on PEP8 checking via the pytest-pep8 plugin and disable normal
+            tests. Same as specifying `--pep8 -k pep8` in `args`.
```

Comment:
Here and elsewhere, you need double backticks, otherwise the Sphinx build fails looking for references